receiver/elasticsearchreceiver/testdata/expected_metrics/full.json

```diff
@@ -2456,6 +2456,57 @@
                         ]
                      },
                      "unit": "{shards}"
+                  },
+                  {
+                     "description": "The current heap memory usage",
+                     "gauge": {
+                        "dataPoints": [
+                           {
+                              "asInt": "285158912",
+                              "startTimeUnixNano": "1661811689941624000",
+                              "timeUnixNano": "1661811689943245000"
+                           }
+                        ]
+                     },
+                     "name": "jvm.memory.heap.used",
```

Comment:
Why does this appear in this PR? Need a rebase?

Reply:
That's one of the metrics that I've added to be emitted in this PR (see https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/16346/files#diff-9e20fd1298180a1ab0c45e590f71b1e98e893104c458140f5ef744c528f2a960R304-R325).
It is already emitted on node-level, but this is on cluster-level (aggregated by `elasticsearch`). Currently the only way to get this metric is to manually sum by nodes after receiving the data - that's not very convienient.

Reply:
Makes sense. Thanks for explaining.